app/models/host.rb

```diff
@@ -20,8 +20,9 @@ class Host < ActiveRecord::Base
   }
 
   FASTLY_BOUNCER_SERVICE_MAP = %w(151.101.2.30 151.101.66.30 151.101.130.30 151.101.194.30).freeze # bouncer.gds.map.fastly.net.
+  FASTLY_NEW_BOUNCER_IPS = %w(151.101.0.204 151.101.64.204 151.101.128.204 151.101.192.204).freeze
```

Comment:
I'm happy to rename `FASTLY_NEW_BOUNCER_IPS` if you can suggest something more accurate - I don't know what these ones should actually be called :)

Reply:
I think that's fine for now. I have no better thing to call them!